src/main/scripts/run_e2e_test_module.py

```diff
@@ -0,0 +1,115 @@
+# Copyright © 2021 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+import io
+import json
+import os
+import requests
+import subprocess
+import sys
+import xml.etree.ElementTree as ET
+import zipfile
+import shutil
+import xml.dom.minidom
+from typing import List
+
+def run_shell_command(cmd):
+    process = subprocess.run(cmd.split(" "))
+    assert process.returncode == 0
+
+if __name__ == "__main__":
+    # Start CDAP sandbox
+    print("Downloading CDAP sandbox")
+    sandbox_url = "https://builds.cask.co/artifact/CDAP-BUT/shared/build-latestSuccessful/SDK/cdap/cdap-standalone/target/cdap-sandbox-6.8.0-SNAPSHOT.zip"
+    sandbox_dir = sandbox_url.split("/")[-1].split(".zip")[0]
+    r = requests.get(sandbox_url)
+    z = zipfile.ZipFile(io.BytesIO(r.content))
+    z.extractall("./sandbox")
+    print("Start the sandbox")
+    run_shell_command(f"chmod +x sandbox/{sandbox_dir}/bin/cdap")
+    os.system("export JAVA_OPTS=-Xmx16G")
+    run_shell_command(f"sandbox/{sandbox_dir}/bin/cdap sandbox start")
+    
+
+    # Build the plugin
+    os.chdir("plugin")
+
+    doc = xml.dom.minidom.parse("pom.xml")
+    modules = doc.getElementsByTagName("module")
+    arguments_list : List[str] = []
```

Comment:
nit: `arguments_list` -> `modules`

Reply:
Changed the variable name to modules_to_build